### ISSUE TITLE
Fails Simple Union / Intersection

### ISSUE THREAD

**Issue by srt19170** [w8r/martinez#45]
```javascript
        let poly1 = [[0, 0],[22.75, -12],[22.75, 0]];
	let poly2 = [[-2.42, 0],[-2.42, -5.45],[2, 0]];
        let result = Martinez.intersection([poly1], [poly2]);
        // result == null
```
Union also fails, returning the two original polygons.

**Comment by w8r** [comment]
It's programmed behaviour for a trivial case. Probably very easy to fix.

**Comment by srt19170** [comment]
I guess that's good to hear :-)  Is there any workaround or anything I can do in the meantime?  This isn't really my wheelhouse, but if you can point me at the right bit of code maybe I can figure it out.

For what it's worth, it works correctly if the polygons are closed off.  But I find in general that the code works poorly if polygons are closed off, so I assume it expects them to be open?

**Comment by w8r** [comment]
Yes, linear rings are supposed to be closed

Kind regards,
Alexander Milevski

**Comment by w8r** [comment]
works in master, will be fixed after #42
